posts/_markdown/2022_karen_intern_post.md

```diff
@@ -0,0 +1,26 @@
+Weeks 1-4: Full time training with Qubit by Qubit. We learned Quantum Mechanics, Linear Algebra, and Computer Science from our instructor Phil Labrum. We got python interview questions to increase professency. Katherine Hanna and Lana Martin provided the leadership and professional development courses.
+Weeks 5-6: Maria and I worked part-time with the Unitary Fund team and built the first open source Randomized Compiler with a group of my classmates from Qubit by Qubit.
+Weeks 7-10: Full time work with the Unitary fund team.
+
+Metriq: Working at Unitary Fund Part-Time
+
+During my two weeks of part time with Unitary Fund, Maria and I were assigned to work on the Metriq Project with Daniel Strano and Vincent Russo. Metriq is focused on making accessible benchmarks available for anyone in the quantum computing community. We made 10 submissions a week to the website adding tasks, methods, platforms, tags, and results to build the Metriq database as well as submitting issues regarding the website. I also started participating in the Discord and learning wonderful things from such a diverse community. 
+
+Metriq: Working at Unitary Fund Full-Time
+
+After being familiarized with Metriq, while posting issues whenever we deem fit. It was time to work and fix those issues. Since we worked with python during our Qubit by Qubit training, we were assigned to update the python client with the current JSON models. But first, we had to become full stack developers and use git. We learned how git works and how to use it thanks to Dan who guided us through every hurdle in the process. After a week of constant researching and trial and error, I finally had the frontend and backend environments accessible on my computer! I learned from Dan so much throughout that week. Mostly how experience, especially with software environments, betters your problem solving skills but also patience. Being patient with solving a problem but also with yourself giving time for you to learn how to tackle it is key to efficient problem solving. Furthermore, we spent time working on our weekly milestones, solving issues, understanding github etiquette, and communicating ideas and suggestions. We also participated in the majority of community calls. Always attending the UF Engineering meeting to hear the team plan and implement Unitary Fund missions, participating in the Marketing team meeting and learning how to increase accessibility to our projects, and getting to understand more about Mitiq too. Throughout that course of time, we were also able to meet one on one with the team. Talking to Will, Misty, Nathan, Dan, and the rest of the team made it OUR team too. 
```

Comment:
```suggestion
After being familiarized with Metriq, while posting issues whenever we deem fit, it came the time to work and fix those issues. Since we worked with Python during our Qubit by Qubit training, we were assigned to update the Python client with the current JSON models. But first, we had to become full stack developers and use git. We learned how git works and how to use it thanks to Dan who guided us through every hurdle in the process. After a week of constant researching and trial and error, I finally had the frontend and backend environments accessible on my computer! I learned from Dan so much throughout that week. Mostly how experience, especially with software environments, betters your problem solving skills but also patience. Being patient with solving a problem but also with yourself giving time for you to learn how to tackle it is key to efficient problem solving. Furthermore, we spent time working on our weekly milestones, solving issues, understanding github etiquette, and communicating ideas and suggestions. We also participated in the majority of community calls. Always attending the UF Engineering meeting to hear the team plan and implement Unitary Fund missions, participating in the Marketing team meeting and learning how to increase accessibility to our projects, and getting to understand more about Mitiq too. Throughout that course of time, we were also able to meet one on one with the team. Talking to Will, Misty, Nathan, Dan, and the rest of the team made it OUR team too. 
```